src/cloudant/index.py

```diff
@@ -20,7 +20,7 @@
 import json
 
 from ._2to3 import STRTYPE, iteritems_
-from ._common_util import JSON_INDEX_TYPE
+from ._common_util import JSON_INDEX_TYPE, codify, SEARCH_ANALYZERS
 from ._common_util import TEXT_INDEX_TYPE
 from ._common_util import SPECIAL_INDEX_TYPE
 from ._common_util import TEXT_INDEX_ARGS
```

Comment:
Stylistically it is not consistent to have multiple imports from a module on the same line and then other imports individually imported from that same module.  I suggest changing L23-26 to:

```python
from ._common_util import (
    JSON_INDEX_TYPE, 
    TEXT_INDEX_TYPE , 
    SPECIAL_INDEX_TYPE , 
    TEXT_INDEX_ARGS, 
    SEARCH_ANALYZERS,
    codify
)
```

I think this looks cleaner.